manifests/1.3.0/opensearch-1.3.0.yml

```diff
@@ -26,6 +26,12 @@ components:
     checks:
       - gradle:properties:version
       - gradle:dependencies:opensearch.version
+  - name: ml-commons
+    repository: https://github.com/opensearch-project/ml-commons.git
+    ref: main
+    checks:
+      - gradle:properties:version
+      - gradle:dependencies:opensearch.version: opensearch-ml-plugin
```

Comment:
I dont think you need to add `: opensearch-ml-plugin` after.

Reply:
Got it.

So is the naming `opensearch-ml-plugin` or `opensearch-ml-client`?

Reply:
Just talked to @ylwu-amzn as their project is actually a combination of 4 subproject, where opensearch-ml-plugin is the top level project with all dependency trees. And opensearch-ml-client is one of the subproject that required to be on maven.

Thanks.